src/vulkan-cts-runner.cc

```diff
@@ -283,6 +298,21 @@ class ProcessBlockState {
   std::size_t test_idx; /* Index in the main test list of the active test. */
 
   void record_result(enum status status) {
+    std::string casename = ctx.test_cases[test_idx];
+    if (ctx.xfails.find(casename) != ctx.xfails.end()) {
+      switch (status) {
+      case FAIL:
+      case CRASH:
```

Comment:
No timeout?

Reply:
I think you want to put timeouts in your skip list, not xfails (otherwise, if the test sneaks in under the limit, you get intermittent fails)